.github/workflows/package.yml

```
@@ -35,7 +35,7 @@ jobs:
       run: mvn -B package --file pom.xml
       
     - name: Upload WAR file as artifact
-      uses: actions/upload-artifact@v4
+      uses: actions/upload-artifact@v3 
```

actionlint (1.7.4)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)